src/py/flwr/server/superlink/state/in_memory_state.py

```diff
@@ -53,10 +54,13 @@ def store_task_ins(self, task_ins: TaskIns) -> Optional[UUID]:
         # Create task_id and created_at
         task_id = uuid4()
         created_at: datetime = now()
+        # Timestamp in seconds with nanosecond resolution
+        pushed_at = time.time_ns() / 1e9
```

Comment:
Doesn't `time.time()` give us that float value?

Reply:
According to https://docs.python.org/3/library/time.html#time.time, and some discussions on some forums, `time.time()` is less precise than `time.time_ns() /1e9`. The precision of `time.time()` is around `1e-6` second. 

Thus, both `time.time()` and `time.time_ns() /1e9` are sufficient for our cases.